modules/serverless-rn-1-24-0.adoc

```diff
@@ -0,0 +1,39 @@
+// Module included in the following assemblies
+//
+// * /serverless/serverless-release-notes.adoc
+
+:_content-type: REFERENCE
+[id="serverless-rn-1-24-0_{context}"]
+= Release notes for Red Hat {ServerlessProductName} 1.24.0
+
+{ServerlessProductName} 1.24.0 is now available. New features, changes, and known issues that pertain to {ServerlessProductName} on {product-title} are included in this topic.
+
+[id="new-features-1.24.0_{context}"]
+== New features
+
+* {ServerlessProductName} now uses Knative Serving 1.3.
+* {ServerlessProductName} now uses Knative Eventing 1.3.
+* {ServerlessProductName} now uses Kourier 1.3.
+* {ServerlessProductName} now uses Knative `kn` CLI 1.3.
+* {ServerlessProductName} now uses Knative Kafka 1.3.
+* The `kn func` CLI plug-in now uses `func` 0.24.
+
+* Init containers support for Knative services is now generally available (GA).
+
+ifdef::openshift-enterprise[]
+* You can now use the cost management service with {ServerlessProductName}.
+endif::[]
+
+[id="fixed-issues-1.24.0_{context}"]
+== Fixed issues
+
+* Integrating {ServerlessProductName} with {SMProductName} causes the `net-istio-controller` pod to run out of memory on startup when too many secrets are present on the cluster.
++
+It is now possible to enable secret filtering, which makes `net-istio-controller` to only consider secrets with a `networking.internal.knative.dev/certificate-uid` label, thus no longer requiring an increased memory limit.
```

Comment:
I'd still suggest:
```
It is now possible to enable secret filtering so that `net-istio-controller` only considers secrets with a `networking.internal.knative.dev/certificate-uid` label. This means that the `net-istio-controller` pod no longer requires an increased memory limit.
```

up to you, though

Reply:
(Went with Neal's suggestion)